app/views/return-requirements/frequency-collected.njk

```diff
@@ -15,12 +17,48 @@
 {% endblock %}
 
 {% block content %}
+{# Error summary #}
+  {% if error %}
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: [
+        {
+          text: "Select how often readings or volumes are collected",
```

Comment:
Can you reuse the error constant for this since that is already set and its the same text?